src/components/Section.js

```diff
@@ -1,8 +1,8 @@
 import { containerSelector } from "../utils/constants";
 
 export default class Section {
-  constructor({ items, renderer }, containerSelector) {
-    this._renderedItems = items;
+  constructor({ items, renderer }) {
+    this._renderedItems = items || [];
```

Comment:
items are not already rendered. use `_items` as the name

Reply:
Needs correction: items are not already rendered. use _items as the name ` this._items = items || [];`